packages/manager/src/utilities/accountCapabilities.ts

```diff
@@ -1,6 +1,26 @@
 import type { AccountCapability } from '@linode/api-v4';
 
 /**
+ * Determines if a feature should be enabled.
+ *
+ * @returns true if the feature is returned from account.capabilities AND if it is explicitly enabled
+ * by a feature flag
+ *
+ * If you need to launch a production feature, but have it be gated,
+ * you would turn the flag *off* for that environment, but have the API return
+ * the account capability.
```

Comment:
```suggestion
 * the account capability. If the feature will never depend on account.capabilites, use Launch Darkly flags directly via the useFlags hook instead of this function.
```
Optional: worth adding based on our discussion thread in Slack?